### PR TITLE
Support app bundles in default apps

### PR DESCRIPTION
**Pull request by QuentinBisson**
Towards https://github.com/giantswarm/giantswarm/issues/23425

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by QuentinBisson** [comment]
https://github.com/orgs/giantswarm/teams/team-honeybadger can you check this PR?

File: service/controller/key/common.go [hunk 1]
```diff
@@ -57,3 +58,7 @@ func ReleaseName(releaseVersion string) string {
 func ReleaseVersion(getter LabelsGetter) string {
 	return getter.GetLabels()[label.ReleaseVersion]
 }
+
+func IsBundle(appName string) bool {
+	return strings.HasSuffix(appName, "-bundle")
```
Comment (kubasobon) [inline service/controller/key/common.go:63]: Will this support the ` security-pack` bundle? It does not have a matching suffix but it is still an app bundle.

Comment (QuentinBisson) [reply service/controller/key/common.go:63]: Same remarks as in cluster-apps-operator. No but the security pack should be named bundle as per the product decision.
Also, this is only for default apps and the security pack is not one of them

**Comment by uvegla** [comment]
> @uvegla because we plan on making this app a default app for all releases and clsuter operator would override that annotation

Ah, okay, thanks! Now I got that thread connected in my head 👍🏻

File: service/controller/key/common.go (hunk 1 above)
Comment (kubasobon) [reply service/controller/key/common.go:63]: I missed that one, thanks for clarifying.

**Comment by QuentinBisson** [comment]
@giantswarm/team-honeybadger can I get an approval here? :D

**Review by uvegla — APPROVED** [review]

**Pull request merged**